modules/nw-metallb-configure-l2-advertisement-label.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-label_{context}"]
+= Configuring MetalLB with an L2 advertisement and label
+
+The `ipAddressPoolSelectors` field in the `bgpAdvertisement` and `l2Advertisement` custom resource defintions is used to associate the IPAddressPool to the advertisement based on the label assigned to the IPAddressPool instead of the name itself. 
```

Comment:
Either bgpadvertisement and l2advertisement, all lowercase OR BGPAdvertisement / L2Advertisement (as in the kind) in case of CRD name